news/2023/2023-12-18-osulazer-updates-preparing-for-ranked-play.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: osu!(lazer) Updates & Preparing For Ranked Play
+date: 2023-12-18 16:00:00 +0000
+---
+
+For this last update of the year, the transition to ranked play in the new client is about to begin its final ascent.
+
+![](/wiki/shared/news/2023-11-30-osulazer-updates-november-30/banner.jpg)
+
+They say learning through *multiple mediums* is optimal. While these short text summaries of the changes in each update serve in large part just to supplement the main video, this time it may be helpful to have detailed explanations both in text and video format. For optimal understanding you may want to **both watch the video and use the summary below as reference**.
```

Comment:
```suggestion
They say learning through *multiple mediums* is optimal. While these short text summaries of the changes in each update serve in large part just to supplement the main video, this time it may be helpful to have detailed explanations both in text and video format. For optimal understanding, you may want to **both watch the video and use the summary below as reference**.
```